quantum/pointing_device/pointing_device.c

```diff
@@ -155,6 +158,12 @@ __attribute__((weak)) void pointing_device_init(void) {
 #    endif
 #endif
     }
+#ifdef POINTING_DEVICE_HIRES_SCROLL_ENABLE
+    hires_scroll_resolution = POINTING_DEVICE_HIRES_SCROLL_MULTIPLIER;
```

Comment:
This isn't defined previously in this file.  So, it will error out. Same with the exponent define.